sys/include/xtimer/implementation.h

```diff
@@ -62,21 +66,22 @@ static inline uint32_t _xtimer_lltimer_mask(uint32_t val)
  * @internal
  */
 uint64_t _xtimer_now64(void);
-int _xtimer_set_absolute(xtimer_t *timer, uint32_t target);
-void _xtimer_set(xtimer_t *timer, uint32_t offset);
-void _xtimer_set64(xtimer_t *timer, uint32_t offset, uint32_t long_offset);
-void _xtimer_periodic_wakeup(uint32_t *last_wakeup, uint32_t period);
-void _xtimer_set_msg(xtimer_t *timer, uint32_t offset, msg_t *msg, kernel_pid_t target_pid);
-void _xtimer_set_msg64(xtimer_t *timer, uint64_t offset, msg_t *msg, kernel_pid_t target_pid);
-void _xtimer_set_wakeup(xtimer_t *timer, uint32_t offset, kernel_pid_t pid);
-void _xtimer_set_wakeup64(xtimer_t *timer, uint64_t offset, kernel_pid_t pid);
+int _xtimer_set_absolute(xtimer_t *timer, uint32_t target_ticks);
+void _xtimer_set(xtimer_t *timer, uint32_t offset_ticks);
+void _xtimer_set64(xtimer_t *timer, uint32_t offset_ticks, uint32_t long_offset_ticks);
+void _xtimer_set64_usec(xtimer_t *timer, uint64_t target_usec);
```

Comment:
rename to xtimer_set64, move to xtimer.h, needs Doxygen (OK to copy docs from xtimer_set and adapt)